cpp/src/parquet/metadata.h

```diff
@@ -188,6 +188,8 @@ class PARQUET_EXPORT FileMetaData {
 
   void WriteTo(::arrow::io::OutputStream* dst) const;
 
+  std::string ToString() const;
```

Comment:
based on the description it sounds like this is returning a binary format value.  I think a more appropriate name would be SerializedMetadata() if it is intended to be passed back to Make above.